utils/env.go

```diff
@@ -95,3 +121,24 @@ func ViamTCPSockets() bool {
 	return runtime.GOOS == "windows" ||
 		slices.Contains(EnvTrueValues, os.Getenv("VIAM_TCP_SOCKETS"))
 }
+
+// LogViamEnvVariables logs the list of viam environment variables in [os.Environ] along with the env passed in.
+func LogViamEnvVariables(msg string, envVars map[string]string, logger logging.Logger) {
+	var env []string
+	for _, v := range os.Environ() {
+		if !strings.HasPrefix(v, ViamEnvVarPrefix) {
+			continue
+		}
+		env = append(env, v)
+	}
+	for key, val := range envVars {
+		// mask the secret
```

Comment:
Will the module still get access to this?

Reply:
ya, the map passed in is not updated